docs/func-op-coverage.md

```diff
@@ -1,11 +1,9 @@
-# Standard Op Coverage
+# Func Op Coverage
 
-The table below shows the supported Standard ops.
+The table below shows the supported func ops.
 
 | op                    | supported          | comment |
 | :-------------------- |:------------------:| :------ |
-| br                    | :white_check_mark: | via `emitc-translate` |
 | call                  | :white_check_mark: | via `emitc-translate` |
-| cond_br               | :white_check_mark: | via `emitc-translate` |
 | constant              | :white_check_mark: | via `emitc-translate` |
```

Comment:
I don't think we support constants for [function symbols](https://mlir.llvm.org/docs/Dialects/Func/#funcconstant-mlirfuncconstantop). Seems to be an overlook from the arith Dialect split.

```suggestion
```

Reply:
We actually do support `func::ConstantOp`, see https://github.com/llvm/llvm-project/blob/406d418c0c831287b41c83150369fd8485c3d700/mlir/lib/Target/Cpp/TranslateToCpp.cpp#L924 and https://github.com/llvm/llvm-project/blob/406d418c0c831287b41c83150369fd8485c3d700/mlir/lib/Target/Cpp/TranslateToCpp.cpp#L240-L246. It might be that there is no full support, but this what came with the dialect renaming llvm/llvm-project@23aa5a7.